configure.ml

```diff
@@ -666,17 +666,15 @@ let natdynlinkflag =
 
 (** * OS dependent libraries *)
 
-let osdeplibs = "-cclib -lunix"
-
-let operating_system, osdeplibs =
+let operating_system =
   if starts_with arch "sun4" then
     let os, _ = run "uname" ["-r"] in
     if starts_with os "5" then
-      "Sun Solaris "^os, osdeplibs^" -cclib -lnsl -cclib -lsocket"
+      "Sun Solaris "^os
```

Comment:
I was wondering if Solaris still existed. Yes, but: https://www.networkworld.com/article/3222707/data-center/the-sun-sets-on-solaris-and-sparc.html